mobile/src/calendar/dayoff-action-button.tsx

```diff
@@ -23,7 +37,18 @@ export class DayoffActionButton extends Component<DayoffActionButtonProps> {
     }
 
     public onDayoffAction = () => {
-        // TODO: do something
+        if (!this.props.interval && this.props.hoursCredit.hours !== 0) {
```

Comment:
what is hoursCredit? The value that we show on top? Why do you check for it being equal to 0 in that case?

Reply:
> what is hoursCredit? The value that we show on top?

it's a model, containg hoursCredit value, and its textual representation - _days + 1/2_. Accordingly, We show textual representation on the top.

Reply:
why do you check on hours !== 0?

Reply:
I thought, 0 means there is neither dayoff nor workout hours, so disable to process.